src/main/java/com/TheAccountant/dto/charge/ChargeDTO.java

```diff
@@ -0,0 +1,77 @@
+package com.TheAccountant.dto.charge;
+
+/**
+ * Created by Florin on 7/30/2017.
+ */
+public class ChargeDTO {
+
+    private String description;
+    private long amount;
+    private Currency currency;
+    private String stripeEmail;
+    private String stripeToken;
+    private Boolean paymentApproved;
```

Comment:
use boolean instead of Boolean. Boolean can be null and we can have NPE situations